handlers/community.js

```diff
@@ -68,6 +68,42 @@ Dota2.Dota2Client.prototype.requestProfileCard = function(account_id, callback)
                     onProfileCardResponse, callback);
 };
 
+/**
+ * @typedef {Object} module:Dota2.schema.k_EMsgClientToGCSetProfileCardSlot
+ * @property {number} slot_id - The slot id to update
+ * @property {number} slot_type - One of {@link module:Dota2.EProfileCardSlotType}
+ * @property {number} slot_value - Depends on the slot_type. For k_EProfileCardSlotType_Empty its ignored.
+ *                                 For k_EProfileCardSlotType_Stat, one of {@link module:Dota2.EStatID}.
+ *                                 For k_EProfileCardSlotType_Hero, it seems to be 4294967296 (32 bit max) + hero id.
+ *                                 For anything else, you should check with NetHook.
+ */
+
+/**
+ * Sends a message to the Game Coordinator updating the logged in user's profile card slots. 
+ * Listen for {@link module:Dota2.Dota2Client#event:profileCardUpdate|profileCardUpdate} event for Game Coordinator's response. 
+ * Requires the GC to be {@link module:Dota2.Dota2Client#event:ready|ready}.
+ * @alias module:Dota2.Dota2Client#setProfileCardSlots
+ * @param {(Dota2.schema.k_EMsgClientToGCSetProfileCardSlot|Dota2.schema.k_EMsgClientToGCSetProfileCardSlot[])} slots - 1 or more profile card slots
+ */
+Dota2.Dota2Client.prototype.setProfileCardSlots = function(slots) {
+    slots = slots || [];
+    slots = (Array.isArray(slots) ? slots : [slots]);
+    
+    if (slots.length == 0) {
+        this.Logger.error("Account ids must be a single id or array of ids.");
+        return null;
+    }
+    
+    this.Logger.debug("Sending request to set profile card slots");
+    
+    var payload = {
+        slots
+    };
+    this.sendToGC(  Dota2.schema.lookupEnum("EDOTAGCMsg").values.k_EMsgClientToGCSetProfileCardSlots, 
+                    Dota2.schema.lookupType("CMsgClientToGCSetProfileCardSlots").encode(payload).finish(), 
+                    onSetProfileCardSlotsResponse);
```

Comment:
Remove this line. Handler is useless without a callback
https://github.com/Arcana/node-dota2/blob/a7e40fffd3d2e8af05ebced4a94afd8ad009d9cf/handlers/helper.js#L181-L187